control-plane/grpc/src/operations/node/server.rs

```diff
@@ -71,4 +72,39 @@ impl NodeGrpc for NodeServer {
             })),
         }
     }
+    async fn cordon_node(
+        &self,
+        request: tonic::Request<CordonNodeRequest>,
+    ) -> Result<tonic::Response<CordonNodeReply>, tonic::Status> {
+        let req: CordonNodeRequest = request.into_inner();
+        match self.service.cordon(req.node_id.into(), req.label).await {
+            Ok(node) => {
+                let resp = Response::new(CordonNodeReply {
+                    reply: Some(cordon_node_reply::Reply::Node(node.into())),
+                });
+                Ok(resp)
+            }
```

Comment:
```suggestion
            Ok(node) => Ok(Response::new(CordonNodeReply {
                    reply: Some(cordon_node_reply::Reply::Node(node.into())),
            })),
```
Can we directly return from here?

Reply:
> Can we directly return from here?

What's the advantage? Just making the code denser often doesn't help - it makes it harder to unpick

Reply:
With the new clippy version, IIRC it'll probably warn if we put it on a variable to simply return it on the next statement.